__fixtures__/example-todo-main/api/src/lib/polyfill.js

```diff
@@ -1 +1,299 @@
-''.replaceAll('a', 'b')
\ No newline at end of file
+/**
+ * Redwood targets Node.js 12, but that doesn't factor into what gets polyfilled
+ * because Redwood uses the plugin-transform-runtime polyfill strategy.
+ *
+ * Also, plugin-transform-runtime is pinned to core-js v3 (no minors, no patches),
+ * so the list of available polyfill is a little outdated.
+ *
+ * Most of these examples are taken from the core-js README, the proposal's README, or MDN.
+ *
+ * Key:
+ * - ✅ -> plugin-transform-runtime successfully polyfills this
+ * - ❌  -> plugin-transform-runtime fails to polyfill this
+ */
```

Comment:
This whole file is from https://github.com/redwoodjs/redwood/pull/6208, unchanged. But I will probably add to it in this PR.